common/awf_vehicle_rviz_plugin/awf_2d_overlay_vehicle/package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>awf_2d_overlay_vehicle</name>
+  <version>0.0.1</version>
+  <description>
+    RViz2 plugin for 2D overlays in the 3D view. Mainly a ROS2 port of the JSK overlay plugin
+        (https://github.com/jsk-ros-pkg/jsk_visualization).
+  </description>
+  <maintainer email="khalil@leodrive.ai">Khalil Selyan</maintainer>
+
+  <license>BSD-3-Clause</license>
+
+  <depend>autoware_auto_vehicle_msgs</depend>
+  <depend>autoware_perception_msgs</depend>
+  <depend>boost</depend>
+  <depend>rosidl_generator_traits</depend>
```

Comment:
```suggestion
```
This prevented me from doing `rosdep install -y --from-paths src --ignore-src --rosdistro $ROS_DISTRO`